matter_server/server/helpers/paa_certificates.py

```diff
@@ -11,24 +11,53 @@
 import logging
 from os import makedirs
 import re
+from typing import List
 
 from aiohttp import ClientError, ClientSession
 from cryptography import x509
 from cryptography.hazmat.primitives import serialization
 
 from matter_server.server.const import PAA_ROOT_CERTS_DIR
 
+# Git repo details
+OWNER = "project-chip"
+REPO = "connectedhomeip"
+PATH = "credentials/development/paa-root-certs"
+
 LOGGER = logging.getLogger(__name__)
 PRODUCTION_URL = "https://on.dcl.csa-iot.org"
 TEST_URL = "https://on.test-net.dcl.csa-iot.org"
-GIT_URL = "https://github.com/project-chip/connectedhomeip/raw/master/credentials/development/paa-root-certs"  # pylint: disable=line-too-long
-GIT_CERTS = [
-    "Chip-Test-PAA-FFF1-Cert",
-    "Chip-Test-PAA-NoVID-Cert",
-]
+GIT_URL = f"https://raw.githubusercontent.com/{OWNER}/{REPO}/master/{PATH}"
+
+
 LAST_CERT_IDS: set[str] = set()
 
 
+async def get_directory_contents(owner: str, repo: str, path: str) -> List[str]:
+    """
+    Fetch directory contents from a GitHub repository.
+
+    Args:
+        owner (str): The owner of the GitHub repository.
+        repo (str): The name of the GitHub repository.
+        path (str): The path within the repository.
+
+    Returns:
+        list: A list of file names in the specified directory.
+    """
+    api_url = f"https://api.github.com/repos/{owner}/{repo}/contents/{path}"
+    async with ClientSession() as session:
```

Comment:
I suggest we inline this inside `fetch_git_certificates` since we already use a client session there. We can use the same session.

Reply:
Ack. Made the requested change